### ISSUE TITLE
[a11y] Aria label property on dialog role

### ISSUE THREAD

**Issue by Palgie** [vue-final/vue-final-modal#208]
Hi - Firstly, thanks for your work on this project! 🙂

Since [axe-core 4.3](https://dequeuniversity.com/rules/axe/4.3/) elements that are classed as ARIA dialog and alertdialog must have an accessible name.

Currently there is no way for users of the plugin to modify the parent component to influence the aria-label for the parent modal. Because of this AXE automation or tests that occur during the modal being open will result in a 'serious' accessibility failure being alerted.

I've patched this so far by doing approaching the problem with:

```diff
diff --git a/node_modules/vue-final-modal/lib/VueFinalModal.vue b/node_modules/vue-final-modal/lib/VueFinalModal.vue
index 6afd3b4..a036c8e 100644
--- a/node_modules/vue-final-modal/lib/VueFinalModal.vue
+++ b/node_modules/vue-final-modal/lib/VueFinalModal.vue
@@ -39,6 +39,7 @@
         :aria-expanded="visibility.modal.toString()"
         role="dialog"
         aria-modal="true"
+        :aria-label="ariaLabel"
         tabindex="-1"
         @mouseup.self="onMouseupContainer"
         @mousedown.self="onMousedown"
@@ -127,6 +128,7 @@ export default {
     name: { type: String, default: null },
     value: { type: Boolean, default: false },
     ssr: { type: Boolean, default: true },
+    ariaLabel: { type: String, default: 'modal'},
     classes: CLASS_PROP,
     overlayClass: CLASS_PROP,
     contentClass: CLASS_PROP,
```

It would be great if something similar could be added to the component to support this natively.

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by hunterliu1003** [comment]
Hi @Palgie,
Because of VueFinalModal don't have any content by default. The vfm container attributes already has the `role="dialog"` and `aria-modal="true"`. I think the `ariaLabel` should be added in the slot or the content of the VueFinalModal.
```html
 <div
        ref="vfmContainer"
        role="dialog"
        aria-modal="true"
        tabindex="-1"
        ...
>
        ...
</div>
```

**Comment by arknu** [comment]
I have created a PR to implement this: #372
